jquery.flot.navigate.js

```diff
@@ -117,15 +120,24 @@ can set the default in the options.
             }
         }
 
-        var SNAPPING_CONSTANT = $.plot.uiConstants.SNAPPING_CONSTANT;
-        var PANHINT_LENGTH_CONSTANT = $.plot.uiConstants.PANHINT_LENGTH_CONSTANT;
-
         function onMouseWheel(e, delta) {
             e.preventDefault();
             onZoomClick(e, delta < 0);
             return false;
         }
 
+        function onMouseDown(e) {
+            canDrag = true;
+        }
+
+        function onMouseUp(e) {
+            canDrag = false;
+        }
+
+        function isLeftMouseButtonPressed(e) {
+            return e.button === 0;
```

Comment:
I'm not sure this is valid for all browsers. Both your branches look old, so, if possible, it would be nice to try to reproduce the issue on master and eventually apply the fix on top of it.

Reply:
This is apparently supported on all standards-compliant versions of browsers:

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button